src/components/Combobox/__tests__/combobox-interactions.js

```diff
@@ -0,0 +1,92 @@
+import { expect } from "@storybook/jest";
+import {
+  ELEMENT_TYPES,
+  getByLabelText,
+  getByTestId,
+  getByText,
+  clickElement,
+  resetFocus,
+  typeText,
+  interactionSuite,
+  pressNavigationKey
+} from "../../../__tests__/interactions-helper";
+import { queryByText } from "@storybook/testing-library";
+import { getTestId, NAVIGATIONS_COMMANDS } from "utils/test-utils";
+
+async function getSearchElement(canvas) {
+  const comboboxElement = await getByTestId(canvas, ELEMENT_TYPES.COMBOBOX);
+  const searchElement = await getByLabelText(comboboxElement, "Search for content");
+  return { comboboxElement, searchElement };
+}
+
+async function onTypeFilterComboboxOptionsTest(canvas) {
+  const { comboboxElement, searchElement } = await getSearchElement(canvas);
+  await typeText(searchElement, "jjj");
+  expect(queryByText(comboboxElement, "Option 1")).toBeNull();
+}
+
+async function onSelectExistFilterClearsFilterTest(canvas) {
+  const { comboboxElement, searchElement } = await getSearchElement(canvas);
+  await typeText(searchElement, "jjj");
+  const cleanSearchButton = getByTestId(
+    comboboxElement,
+    getTestId(ELEMENT_TYPES.CLEAN_SEARCH_BUTTON, "combobox-search")
+  );
+  await clickElement(cleanSearchButton);
+  expect(searchElement).toHaveValue("");
+  const option1 = getByText(comboboxElement, "Option 1");
+  expect(option1).toBeTruthy();
```

Comment:
WDYT about
```suggestion
  expect(option1).toBeInTheDocument();
```
or:
```
expect(option1).toBeVisible();
```

I think it's more "accurate"